crates/weaver_resolver/data/registry-test-4-events/registry/browser-event.yaml

```diff
@@ -0,0 +1,54 @@
+groups:
```

Comment:
This is testing that the resolved schema is handling ALL of the existing Semconv defined events as well as `body` and `fields` definition.

Hence, all of the files added to the `registry` folder